### ISSUE TITLE
Angular 5 support

### ISSUE THREAD

**Issue by bohdyone** [owsolutions/amazing-time-picker#62]
Hi,

Nice work on the timepicker.

Is support for Angular 5.2+ currently in the works? At the moment I get an error when compiling:

```
webpack: Compiled with warnings.
webpack: Compiling...
ERROR in node_modules/@angular/core/src/render3/ng_dev_mode.d.ts(9,11): error TS2451: Cannot redeclare block-scoped variable 'ngDevMode'.
node_modules/amazing-time-picker/node_modules/@angular/core/src/render3/ng_dev_mode.d.ts(9,11): error TS2451: Cannot redeclare block-scoped variable 'ngDevMode'.
```

**Comment by torabian** [comment]
Thank you for reporting this. Due to:
https://github.com/angular/angular/issues/21670

It appears to me that it's a possible angular 5.2 problem itself yet. I'll try more investigations about it and notify you.

**Comment by gmfun** [comment]
`@angular/core` and other angular packages are added as dependcies, I think it should in peer dependencies. Otherwise 2 versions of angular might get created. Probably this is the issue

**Comment by puneetkaurbains** [comment]
I agree with @gmfun if you change dependencies to peer dependencies in package.json and publish new version of library it will solve the problem. I need to use this library in my project but because @angular/core are provided as dependencies its creating 2 version of angular and causing issue. Please look into it. Thanks.
